gcsweb/passwd

```diff
@@ -0,0 +1 @@
+nobody:x:65534:65534:nobody:/nonexistent:/usr/sbin/nologin
```

Comment:
Why not just echo this in Dockerfile - one less file in git? :)

Reply:
The distroless image has no echo command. Also, running commands inside Dockerfiles gets trickier with multiarch (which is why I moved the `chmod` to the Makefile).